### DISCUSSION TITLE
Is it possible for a Nova addon to depend on a Spigot plugin?

### DISCUSSION THREAD

**Discussion by anyothn** [xenondevs/Nova#475] [Q&A] [answered]
I'm trying to integrate into [Plasmo Voice](https://modrinth.com/plugin/plasmo-voice) to create custom audio players using Nova tile entities, though, doing so always leads to a NoClassDefFoundError, even if Plasmo Voice is loaded before the addon. Is there any way to load classes from Spigot plugins other than Nova? If so, how? Thanks for the help!

**Comment by NichtStudioCode** [comment, marked as answer]
There is currently no (proper) way to interact with other plugins from a Nova addon, since Nova is loaded as a paper plugin where the class loaders of different plugins are isolated from each other. The plan is to load addons as plugins in the future to resolve this issue.
